resources/deployments/controller/controller_deployments.go

```diff
@@ -297,6 +299,24 @@ func ParseLookupQuery(vals url.Values) (deployments.Query, error) {
 		query.SearchText = search
 	}
 
+	createdBefore := vals.Get("created_before")
+	if createdBefore != "" {
+		if createdBeforeTime, err := parseEpochToTimestamp(createdBefore); err != nil {
+			return query, err
```

Comment:
error message won't contain from which param issue comes from, use errors.Wrap to deal with messaging